src/main/java/graphql/schema/impl/SchemaUtil.java

```diff
@@ -96,4 +103,29 @@ public static void replaceTypeReferences(GraphQLSchema schema) {
         SchemaTraverser schemaTraverser = new SchemaTraverser(schemaElement -> schemaElement.getChildrenWithTypeReferences().getChildrenAsList());
         schemaTraverser.depthFirst(new GraphQLTypeResolvingVisitor(typeMap), roots);
     }
+
+    public static GraphQLObjectType getOperationRootType(GraphQLSchema graphQLSchema, OperationDefinition operationDefinition) {
```

Comment:
Add util method here. Let me know if there's a better util to chuck this method into

Reply:
Fair enough place.